SygicMapsKit/Classes/SearchModule/SYMKSearchViewController.swift

```diff
@@ -74,19 +85,29 @@ public class SYMKSearchViewController: SYMKModuleViewController {
     }
     
     // MARK: - Private methods
+    
+    private func search(for query: String) {
+        model.search(with: query) { [weak self] (results, state) in
```

Comment:
ech asi search(with:) nie search(for:) my bad :) nech to je jednotne... ale asi to nie je dobre len to search tam nechat... nevadi dorobis s documetaciou